katas/content/nonlocal_games/chsh_quantum_bob_strategy/index.md

```diff
@@ -0,0 +1,8 @@
+**Input:**
+
+- Bob's starting bit (Y)
+- Bob's half of Bell pair he shares with Alice.
+
+**Goal:**
+Measure Bob's qubit in the $\frac{\pi}{8}$ basis if his bit is 0 (false), or the $-\frac{\pi}{8}$ basis if his bit is 1 (true), and return the result.
```

Comment:
We haven't defined the angular basis in this kata yet, so I'm not sure the learner will be able to follow what we're asking them to do. We should include the definition of the basis here, probably by spelling out the basis states that form it explicitly and their mapping to the measurement outcomes (we do something similar in the Measurements in Single-Qubit Systems kata, task "Distinguishing Orthogonal States: 1")

Reply:
Hint is added and Solution is updated (pattern is borrowed from "Distinguishing Orthogonal States: 1" )